openproblems/tasks/denoising/methods/alra.py

```diff
@@ -45,9 +46,60 @@ def alra(adata, test=False):
                 raise
 
     # transform back into original space
+    # functions are reversed!
```

Comment:
This looks like a mistake. You will get better results if you libnorm -> sqrt -> denoise -> square -> denorm.

Reply:
Bumping this.